qutebrowser/mainwindow/statusbar/progress.py

```diff
@@ -40,6 +40,9 @@ class Progress(QProgressBar):
             border: 2px solid transparent;
             margin-left: 1px;
             background-color: transparent;
+            font-size: 4px; /* force very small s.t. it does not affect size hint */
+                            /* if you choose the value too small, it does not */
+                            /* appear when restoring the session */
```

Comment:
When removing the size hint, the progressbar gets quite huge. When setting the sizehint to something like `QSize(90,4)`, one has the same effect as before (that one in the image above). But overriding `minimumSizeHint()` works if one keeps the `self.fontMetrics().height()` without adding additional height.

I've just updated the commit of the PR.